src/main/webapp/resourses/application.properties

```diff
@@ -0,0 +1,15 @@
+reports-path=C:/Users/ivan.shastakou/IdeaProjects/employeestat/reports
+reports-generating-cron=0 0 0 1 * *
+
+security.oauth2.client.clientId=854909843289-i6kivqbpjvt6di44ur0f0p2li5novibf.apps.googleusercontent.com
+security.oauth2.client.clientSecret=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
```

Comment:
never store sensitive data like real secrets and passwords in git. It can be stolen and used by intruders. 
If we speak about microservices, secrets are usually stored in some vault, which is part of infrastructure, so microservice can read secrets from it when deployed.
For legacy stack secrets are stored in some local vaults or shared between team somehow